### ISSUE TITLE
Targeted Request in postman.setNextRequest() can't set Environment Variables

### ISSUE THREAD

**Issue by windupurnomo** [a85/POSTMan-Chrome-Extension#4608]
I use `postman.setNextRequest("login")` to call login request. Inside login test I set environment variables from token login response. It's not work. Login testing is success, but environment variable token not exists.

_Pre Request Script in Collection:_
``` javascript
var email = "foo@bar.com";
var password = "mypassword";
pm.environment.set("email", email); // success to set env variable
pm.environment.set("password", password); // success to set env variable
postman.setNextRequest("login");
postman.setNextRequest(null);
```

_Test Script in login request:_
``` javascript
pm.test("Status code is 200", function () {
    pm.response.to.have.status(200);
    var dt = pm.response.json();
    pm.environment.set("token", dt.token); // failed to set env variable
});

**Comment by sivcan** [comment]
@windupurnomo Couple of questions:
1. Which Postman version are you using and what OS?
2. Is 'Persist Variables' checkbox enabled before running the request? 
3. Is the environment selected from the dropdown before running the collection?

I tried to reproduce this issue but couldn't. 
Can you also console your object variable `dt` and also check if it does have the value `token`?
If it's undefined then the variable will be removed from the environment when using `pm.environment.set`

**Comment by windupurnomo** [comment]
_1. Which Postman version are you using and what OS?_
I use 6.0.10 for MacOS

_2. Is 'Persist Variables' checkbox enabled before running the request?_
No, I don't check 'Persist Variables'

_3. Is the environment selected from the dropdown before running the collection?_
Yes, I selected environment variable...

Your question number 2, resolve my problem.. 
Now, I check 'Persist Variable' checkbox and I got that I want.. 
thank you @sivcan
